app/controllers/registration_controller.rb

```diff
@@ -216,7 +216,8 @@ def validate_list_admin
 
   def list_admin
```

Comment:
Do we have tests that check if list_admin returns the correct data?

Reply:
Great point! Tests added.